### PR TITLE
Changed package to noarch

### PR DESCRIPTION
**Pull request by sbluhm**
Changed package to "noarch". This omits trying to build a debug package which would fail on EL builds due to missing binaries.

### CODE REVIEW

**Comment by sbluhm** [comment]
@renner /@juliogonzalez can you please review/merge?

**Comment by juliogonzalez** [comment]
Adding reviewers. Change looks OK, but I am not sure if there are unintended consequences :-)

**Review by witekest — APPROVED** [review]
Looks good to me. I think we don't need version number bump here.

**Review by aaannz — APPROVED** [review]
👍🏾 
I am not sure how is the release procedure here. But if this is similar to retail, I am not sure if version bump is not required as well. Sideeffect having spec in git and using obs  _service.

**Comment by aaannz** [comment]
@agraul Any thought?

File: openvpn-formula/openvpn-formula.spec [hunk 1]
```diff
@@ -19,6 +19,7 @@
 Name:           openvpn-formula
 Version:        0.1.1
```
Comment (agraul) [inline openvpn-formula/openvpn-formula.spec:20]:
```suggestion
Version:        0.1.2
```

File: openvpn-formula/openvpn-formula.changes [hunk 2]
```diff
@@ -1,3 +1,8 @@
+-------------------------------------------------------------------
+Fri Jul 16 07:36:14 UTC 2021 - Stefan Bluhm <stefan.bluhm@clacee.eu>
+
+- Changed package to noarch.
```
Comment (agraul) [inline openvpn-formula/openvpn-formula.changes:4]:
```suggestion
- Update to v0.1.2
- Changed package to noarch.
```

**Review by agraul — CHANGES_REQUESTED** [review]
> @agraul Any thought?

You're right, we to bump the spec to 0.1.2 and `git tag` it with `openvpn-v0.1.2` after merging  this PR. 

I left suggestions that do the changes for the spec and the changlog file inline.

**Review by sbluhm — COMMENTED** [review]
Done

**Review by agraul — APPROVED** [review]
Thanks! I will merge, tag and create a SR later.

**Pull request merged**